src/dvc_render/vega.py

```diff
@@ -120,5 +121,8 @@ def generate_markdown(self, report_path=None) -> str:
             plt.savefig(output_file)
             plt.close()
 
-            return f"\n![{self.name}]({output_file.relative_to(report_folder)})"
```

Comment:
I think this actually breaks the integration with CML like:

https://github.com/iterative/example-get-started-experiments/blob/main/.github/workflows/run-studio-experiment.yml#L61

Could we make the base64 optional, default to the previous behavior, and explicitly enable in the DVCLive side only for notebook?

Reply:
Updated to make the file optional and to return the markdown string if no file is passed. PTAL.

Reply:
Updating dvclive to use this option. Should work the same as before as long as a file is passed.